config/initializers/devise.rb

```diff
@@ -266,7 +266,7 @@
   # config.navigational_formats = ['*/*', :html]
 
   # The default HTTP method used to sign out a resource. Default is :delete.
-  config.sign_out_via = :delete
+  config.sign_out_via = %i[delete get]
```

Comment:
hmmmmm

Why is this required?

For the `Sign out` button/link?

You can always enforce links to use `delete` method if required.
I don't see a problem on using `get` requests, but wanted to callout that 😄

Reply:
Oh no , it was to test sign out via URL (before adding an index page and the link)  but later I forgot to add delete method in the link. Thanks